src/emulator/registry.ts

```diff
@@ -92,6 +98,11 @@ export class EmulatorRegistry {
   }
 
   static isRunning(emulator: Emulators): boolean {
+    if (emulator === Emulators.EXTENSIONS) {
+      return (
+        this.extensionsEmulatorRegistered && this.INSTANCES.get(Emulators.FUNCTIONS) !== undefined
```

Comment:
wdyt of this to avoid relying on implementation detail? 
```
        this.extensionsEmulatorRegistered && this.isRunning(Emulators.FUNCTIONS)
```

Reply:
Ahh, that's probably cleaner. I had automerge on, so I'll make another PR to clean this up